.github/workflows/azure-deploy.yml

```
@@ -0,0 +1,186 @@
+jobs:
+  deploy:
+    if: github.repository == 'digitaldemocracy2030/kouchou-ai'
+    runs-on: ubuntu-latest
+    environment: production
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Azure CLI ログイン
+        uses: azure/login@v1
```

actionlint (1.7.7)

the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)